CHANGELOG.md

```diff
@@ -13,18 +13,24 @@ For details about compatibility between different releases, see the **Commitment
 
 ### Changed
 
-- Gateway ID and device ID are not autogenerated and prefilled anymore.
-
 ### Deprecated
 
 ### Removed
 
 ### Fixed
 
-- Fixed an issue where an downlink message was scheduled by the Application Layer Clock Synchronization (ALCS) implementation when there is no answer to send (i.e. `AnsRequired` is not set and the difference with what the end device reports falls within the threshold).
-
 ### Security
 
+## [3.31.1] - unreleased
+
+### Changed
+
+- Gateway ID and device ID are not autogenerated and prefilled anymore.
+
+### Fixed
+
+- Fixed an issue where an downlink message was scheduled by the Application Layer Clock Synchronization (ALCS) implementation when there is no answer to send (i.e. `AnsRequired` is not set and the difference with what the end device reports falls within the threshold).
```

Comment:
We backported this to the previous release right?

Reply:
I think you are right, but the changelog entry was added in a wrong place
fix PR https://github.com/TheThingsNetwork/lorawan-stack/pull/7154/files
changelog entry https://github.com/TheThingsNetwork/lorawan-stack/commit/05c778c7a5813cd3c76ef59143d7f3919779492c

Reply:
Or rather it was not moved when the change was backported